modules/apps/27-interchain-accounts/genesis/types/genesis.go

```diff
@@ -90,7 +91,7 @@ func (gs ControllerGenesisState) Validate() error {
 // DefaultHostGenesis creates and returns the default interchain accounts HostGenesisState
 func DefaultHostGenesis() HostGenesisState {
 	return HostGenesisState{
-		Port:   PortID,
+		Port:   icatypes.PortID,
 		Params: hosttypes.DefaultParams(),
```

Comment:
Sorry for just noticing this, but maybe we can just move this function to hosttypes? Same for controller types. I think that should remove the host/controller imports

Reply:
but then we still need to import controllertypes/hosttypes into genesis types to use the function, which if we leave the genesis in ica will trigger the original import cycle issue

Reply:
Oh yea because this is going to be called by the `DefaultGenesis()` function